python/paddle/v2/fluid/executor.py

```diff
@@ -46,10 +46,11 @@ def as_numpy(tensor):
     assert isinstance(tensor, core.LoDTensor)
     lod = tensor.lod()
     tensor_data = np.array(tensor)
-    if len(lod) == 0:
-        ans = tensor_data
-    else:
-        raise RuntimeError("LoD Calculate lacks unit tests and buggy")
+    ans = tensor_data
+    # if len(lod) == 0:
+    #    ans = tensor_data
+    # else:
+    #    raise RuntimeError("LoD Calculate lacks unit tests and buggy")
```

Comment:
I saw an related issue #8038 . #8038 seems failed here. I am not quite familiar with the codes. @kexinzhao could you please explain some about this? @qingqing01 could you have a look?

Reply:
> raise RuntimeError("LoD Calculate lacks unit tests and buggy")

The origin interface does not support to fetch LoDTensor, whose LoD is not empty. 

```python
    def run(self,
            program=None,
            feed=None,
            fetch_list=None,
            feed_var_name='feed',
            fetch_var_name='fetch',
            scope=None,
            return_numpy=True):
```

So users can specify `return_numpy=Fale` to get the variable, not numpy array. The users can get numpy.array by ` np.array(var) `

Reply:
Thanks @qingqing01 for the explanation. 
Here I temporarily change the code in `as_numpy` so that I can get a numpy array to print its `shape`. 
I will revert this change and get the lod info of the output LoDTensor and convert it to `np.array` in the test code instead.

Reply:
Done